config/initializers/cors.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
```

Comment:
- [ ] The previous reviewer suggested that you add the `rack-cors` gem in the Gemfile and then add this file. But from your commit history, I can see that you added the gem and later removed it. Now I am unable to run your app due to an error. Kindly add the gem to the gem file as suggested to avoid errors when setting up the project.

  ![image](https://github.com/aubinleyoung/hello-rails-back-end/assets/54848363/67a5c8f9-934a-4869-b439-9e5063f0577f)

Reply:
Thank you @DammyShittu  for your feedback, I used the command `gem install rack-cors` to install the rack-cors, that's why I removed it from the Gemfile.